libraries/redcore/toolbar/builder.php

```diff
@@ -194,18 +194,18 @@ public static function createCancelButton($task, $class = '')
 	/**
 	 * Create a close button.
 	 *
-	 * @param   string  $task   The task name.
-	 * @param   string  $class  A css class to add to the button.
+	 * @param   string  $task      The task name.
+	 * @param   string  $class     A css class to add to the button.
+	 * @param   bool    $isOption  Is button a menu option?
 	 *
 	 * @return  RToolbarButtonStandard  The button.
 	 */
-	public static function createCloseButton($task, $class = '')
+	public static function createCloseButton($task, $class = '', $isOption = false)
```

Comment:
one idea...

what if we make it like this...



	/**
	 * Create a new button.
	 *
	 * @param   string  $task        The task name.
	 * @param   string  $class       A css class to add to the button.
	 * @param   string  $classAddon  A css class add-on.
	 *
	 * @return  RToolbarButtonStandard  The button.
	 */
	public static function createNewButton($task, $class = '', $classAddon = 'btn-success')
	{
		$class = !empty($class) ? $class . ' ' . $classAddon : $classAddon;

		return new RToolbarButtonStandard('JTOOLBAR_NEW', $task, $class, 'icon-file-text', '', false);
	}